tests/unit/src/test/resources/foldingRange/expect/Blocks.scala

```diff
@@ -17,25 +17,25 @@ class A >>region>>{
 
   def longSingleLineBlock = { ???; ???; ???; }
 
-  def chain = >>region>>Seq(1).map>>region>>{
+  def chain =>>region>> Seq(1).map{
     x =>
-    x + 1
-     + 1
+    >>region>>x + 1
```

Comment:
Why do we have a new region here? It's not a definition.

Reply:
Okay so should we still keep the old if statement about the distance to enclosing fold?
```
case Foldable((pos, adjust))
             if span(enclosing) - span(pos) > distanceToEnclosingThreshold =>
```

Just make it not filter out definitions?

Reply:
It's like this now